### PR TITLE
Upgrade dev dependencies

### PR DESCRIPTION
**Pull request by alexdavid**
@charlierudolph @kevgo

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,14 +39,14 @@
   },
   "scripts": {
     "build": "gulp",
-    "feature-tests": "./node_modules/.bin/cucumber-js test/feature --format=progress",
+    "feature-tests": "cucumber-js test/feature --format=progress",
```
Comment (kevgo) [inline package.json:42]: Any reasons for taking this out? If we leave that path in, nobody gets hurt, and it works for more people.

Comment (alexdavid) [inline package.json:42]: npm automatically adds `./node_modules/.bin` to the path of the shell that scripts run in. This does nothing but clutter up package.json

Comment (kevgo) [inline package.json:42]: Really? Why do I have to explicitly add it to my path then?

Comment (alexdavid) [inline package.json:42]: So that you can call them from your shell

npm runs these scripts in its own environment

Comment (kevgo) [inline package.json:42]: Cool, thanks, makes sense!

**Comment by kevgo** [comment]
:ship:

**Comment by charlierudolph** [comment]
:ship:

**Pull request closed without merging**